libs/src/api/user.js

```diff
@@ -135,6 +135,19 @@ class Users extends Base {
     return []
   }
 
+  /**
+   * Returns the top users for the specified genres
+   * @param {number} limit - max # of items to return
+   * @param {number} offset - offset into list to return from (for pagination)
+   * @param {Object} {Array of genres} - filter by genres ie. "Rock", "Alternative"
+   * @param {Boolean} with_users - If the userIds should be returned or the full user metadata
+   * @returns {Object} {Array of user objects if with_users set, else array of userIds}
+   */
+  async getTopAristByGenres (genres, limit = 30, offset = 0, withUsers = false) {
```

Comment:
artist is spelled wrong here

Reply:
🤦 
Do you think we should avoid using the term artist?
We don't really use that term, maybe I should change it to user.
probably should be `getTopUsersByGenres`

Reply:
Technically our db fields all call it creator, not artist